sa/database.go

```diff
@@ -182,3 +183,10 @@ func initTables(dbMap *gorp.DbMap) {
 	dbMap.AddTableWithName(core.SignedCertificateTimestamp{}, "sctReceipts").SetKeys(true, "ID").SetVersionCol("LockCol")
 	dbMap.AddTableWithName(core.FQDNSet{}, "fqdnSets").SetKeys(true, "ID")
 }
+
+func safeSelectOne(m *gorp.DbMap, holder interface{}, q string, args ...interface{}) error {
+	if strings.ContainsAny(q, "'\"\n\r\\") {
```

Comment:
Is it feasible to flip this and use a whitelist approach instead? Maybe checking for a match on `[\w]+`?

I know this function isn't for user input and perhaps doesn't have to be especially robust but enumerating all the badness is going to be tough. E.g. should `;` be in that list or are we confident there's no way to concatenate statements with Gorp/MariaDB?